switss/certification/certificates.py

```diff
@@ -27,52 +77,27 @@ def generate_farkas_certificate(reach_form, mode, sense, threshold,solver="cbc")
     assert (threshold >= 0) and (threshold <= 1)
 
     farkas_matr,rhs = __get_right_constraint_set(reach_form,mode,sense,threshold)
+    if sense in ["<=", "<"]:
+        # if sense is "<=" or "<", then the certificate condition is Ax >=/> b
+        # therefore, apply the transformation: Ax >=/> b <=> (-A)x <=/< -b
```

Comment:
should be:
`if sense is "<=" or "<", then the certificate condition is Ax <=/< b`
right?

Reply:
Ah, no. I see now. Sorry